psi4/src/psi4/libtrans/integraltransform.cc

```diff
@@ -44,240 +44,221 @@
+    useDPD_ = outputType_ == OutputType::IWLAndDPD || outputType_ == OutputType::DPDOnly;
+    iwlAAIntFile_ = transformationType_ == TransformationType::Restricted ? PSIF_MO_TEI : PSIF_MO_AA_TEI;
+    iwlABIntFile_ = transformationType_ == TransformationType::Restricted ? PSIF_MO_TEI : PSIF_MO_AB_TEI;
+    iwlBBIntFile_ = transformationType_ == TransformationType::Restricted ? PSIF_MO_TEI : PSIF_MO_BB_TEI;
 
     tpdm_buffer_ = 0;
 
     aQT_ = init_int_array(nmo_);
-    if(transformationType_ == Restricted){
+    if (transformationType_ == TransformationType::Restricted) {
         reorder_qt(clsdpi_, openpi_, frzcpi_, frzvpi_, aQT_, mopi_, nirreps_);
         bQT_ = aQT_;
-    }else{
+    } else {
         bQT_ = init_int_array(nmo_);
         reorder_qt_uhf(clsdpi_, openpi_, frzcpi_, frzvpi_, aQT_, bQT_, mopi_, nirreps_);
     }
     // Set up the correlated to Pitzer arrays.  These have to include the occupied core terms, because
     // the reference contributions are already folded into the TPDM.  However, they don't include frozen
     // virtuals
     aCorrToPitzer_ = init_int_array(nmo_);
-    if(transformationType_ != Restricted){
+    if (transformationType_ != TransformationType::Restricted) {
         bCorrToPitzer_ = init_int_array(nmo_);
-    }else{
+    } else {
         bCorrToPitzer_ = aCorrToPitzer_;
     }
 
     int nFzvFound = 0;
     int pitzerCount = 0;
-    for(int h = 0; h < nirreps_; ++h){
+    for (int h = 0; h < nirreps_; ++h) {
         for (int p = 0; p < mopi_[h]; p++) {
             if (p < mopi_[h] - frzvpi_[h]) {
                 // This is active, count it
                 int q = aQT_[pitzerCount];
                 aCorrToPitzer_[q] = pitzerCount - nFzvFound;
-                if(transformationType_ != Restricted){
+                if (transformationType_ != TransformationType::Restricted) {
                     int q = bQT_[pitzerCount];
                     bCorrToPitzer_[q] = pitzerCount - nFzvFound;
                 }
-            }else{
+            } else {
                 nFzvFound++;
             }
             pitzerCount++;
         }
     }
 
-    if(print_ > 4){
-        outfile->Printf( "\tThe Alpha Pitzer to QT mapping array:\n\t\t");
-        for(int p = 0; p < nmo_; ++p)
-            outfile->Printf( "%d ", aQT_[p]);
-        outfile->Printf( "\n");
-        outfile->Printf( "\tThe Beta Pitzer to QT mapping array:\n\t\t");
-        for(int p = 0; p < nmo_; ++p)
-            outfile->Printf( "%d ", bQT_[p]);
-        outfile->Printf( "\n");
-        outfile->Printf( "\tThe Alpha Correlated to Pitzer mapping array:\n\t\t");
-        for(int p = 0; p < nmo_; ++p)
-            outfile->Printf( "%d ", aCorrToPitzer_[p]);
-        outfile->Printf( "\n");
-        outfile->Printf( "\tThe Beta Correlated to Pitzer mapping array:\n\t\t");
-        for(int p = 0; p < nmo_; ++p)
-            outfile->Printf( "%d ", bCorrToPitzer_[p]);
-        outfile->Printf( "\n");
+    if (print_ > 4) {
```

Comment:
I think you need braces around the loops here if you are going to format this section

Reply:
Fixed.